wgengine/magicsock/magicsock.go

```diff
@@ -2844,9 +2850,18 @@ func (c *RebindingUDPConn) localAddrLocked() *net.UDPAddr {
 	return c.pconn.LocalAddr().(*net.UDPAddr)
 }
 
+var errNilPConn = errors.New("nil pconn")
```

Comment:
document this a bit? this this never makes its way to end users and is tested for?